sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -664,10 +664,10 @@ object RapidsConf {
     if (asTable) {
       println("")
       // scalastyle:off line.size.limit
-      println("""## Fine Tuning
-        |_The RAPIDS Accelerator for Apache Spark_ can be further configured to enable or disable
-        |specific expressions and to control what parts of the query execute using the GPU or
-        |the CPU.
+      println("""## Supported GPU Operators and Fine Tuning 
+        |_The RAPIDS Accelerator for Apache Spark_ can be configured to enable or disable specific 
+        |GPU accelerated expressions.  Enabling or disabling an expression controls which operators 
```

Comment:
```suggestion
        |GPU accelerated expressions. Enabling or disabling an expression controls which operators 
```

Reply:
Instead of:

> Enabling or disabling an expression controls which operators run on the CPU or GPU. 

This is a suggestion for slightly different wording:

> Enabled expressions are candidates for GPU execution. If the expression is configured as disabled, the accelerator plugin will not attempt replacement, and it will run on the CPU.

Reply:
Updated per recommendation.

Reply:
build